rb/lib/selenium/webdriver/remote/bridge.rb

```diff
@@ -82,7 +82,7 @@ def session_id
         def browser
           @browser ||= begin
             name = @capabilities.browser_name
-            name ? name.tr(' ', '_').downcase.to_sym : 'unknown'
+            name ? name.tr(' -', '_').downcase.to_sym : 'unknown'
```

Comment:
doesn't `#tr` only take one character? I think we need to chain the `#tr` or use `#gsub`.

Reply:
It can take more than one character. One example on the documentation is:

```rb
'hello'.tr('aeiou', '-') # => "h-ll-"
```
> If replacements is shorter than selector, it is implicitly padded with its own last character:

Docs: https://ruby-doc.org/3.2.2/String.html#method-i-tr